src/components/QuizzPage/QuizzPage.jsx

```diff
@@ -11,6 +11,12 @@ const QuizzPage = ({ data, results }) => {
     score: 0,
   });
 
+  // ########## SUGGESTION ##########
+  const FormTitles_SUGGESTION = [
+    ...data.map((entry) => entry.question),
+    "Enter your name: ",
+  ];
```

Comment:
I recommend spreading out the questions via Array.map(). Avoiding using indexing directly. As it can be prone to errors.